doc/how_to/load_your_data_into_sorting.rst

```diff
@@ -0,0 +1,152 @@
+
+
+This :code:`Sorting` contains important information about your spike trains including:
+
+  * spike times: the peaks of the extracellular potentials expressed in samples/frames these can
+    be converted to seconds under the hood using the sampling_frequency
+  * spike labels: the neuron id for each spike, can also be called cluster ids or unit ids
+    Stored as the :code:`unit_ids` in SpikeInterface
+  * sampling_frequency: the rate at which the recording equipment was run at. Note this is the
+    frequency and not the period. This value allows for switching between samples/frames to seconds
+
+
+There are 3 options for loading your own data into a sorting object
+
+With lists of spike trains and spike labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this case we need a list of spike times unit labels, sampling_frequency and optional unit_ids
+if you want specific labels to be used (in this case we only create the :code:`Sorting` based on
+the requested unit_ids).
+
+.. code-block:: python
+
+    import numpy as np
+    from spikeinterface.core import NumpySorting
+
+    # in this case we are making a monosegment sorting
+    # we have four spikes that are spread among two neurons
+    my_sorting = NumpySorting.from_times_labels(
+                                                times_list=[
+                                                    np.array([1000,12000,15000,22000])   # Note these are samples/frames not times in seconds
+                                                    ],
+                                                labels_list=[
+                                                    np.array(["a","b","a","b"])
+                                                    ],
+                                                sampling_frequency=30_000.0
+                                                )
+
+
+With a unit dictionary
+^^^^^^^^^^^^^^^^^^^^^^
+
+We can also use a dictionary where each unit is a key and its spike times are values.
+This is entered as either a list of dicts with each dict being a segment or as a single
+dict for monosegment. We still need to separately specify the sampling_frequency
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    my_sorting = NumpySorting.from_unit_dict(
+                                             units_dict_list={
+                                                              '0': [1000,15000],
+                                                              '1': [12000,22000],
+                                                              },
+                                             sampling_frequency=30_000.0
+                                           )
+
+
+With Neo SpikeTrains
+^^^^^^^^^^^^^^^^^^^^
+
+Finally since SpikeInterface is tightly integrated with the Neo project you can create
+a sorting from :code:`Neo.SpikeTrain` objects. See :doc:`Neo documentation<neo:index>`` for more information on
```

Comment:
There's a double ` at the end of the the Neo link, which breaks it. Should be
```
:doc:`Neo documentation<neo:index>`
```

Reply:
Fixed.

Reply:
I also want to say that this was completely the IDE's fault :P